arabot/modules/waifus.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+import random
+from itertools import chain
+
+from aiohttp import ClientResponseError
+from arabot.core import Ara, Category, Cog, Context
+from arabot.utils import AnyMember, humanjoin
+from disnake import Embed
+from disnake.ext.commands import CogMeta, Command
+from waifu import APIException, ImageCategories, WaifuAioClient
+
+REACTION_MAPPING: dict[str, tuple[str, str]] = {
+    "bite": ("{author} wants to bite someone...", "{author} bites {target}"),
+    "blowjob": ("{author} wants to give someone a blowjob...", "{author} gives {target} a blowjob"),
+    "blush": ("{author} is blushing", "{target} make{s} {author} blush"),
```

Comment:
I've been thinking about some cases, such as "blush" where the author makes someone else blush. Isn't it more like the author is blushing?

Same for crying, where to me it feels more appropriate that the author is trying to express their despair, like `{author} is crying on {target}'s shoulders`.

Also smiling, instead of expressing how the target makes the author smile, it feels more natural to say `{author} smiles at {target}`, no?

Reply:
Didn't understand the part about blushing. Author blushes in both cases.

I'll change cry and smile.

Reply:
Changed in 714153f